files/perfect-scrollbar/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "perfect-scrollbar",
+  "repo": "noraesae/perfect-scrollbar",
+  "files": {
+    "include": ["dist/**/*.js", "dist/**/*.css"]
```

Comment:
This config wont retain the directory names. Both js and css files will be added to root. 
Is that correct?